src/index.js

```diff
@@ -135,12 +145,16 @@ async function run() {
           const fileExtension = file.split('.').pop()
           core.info(`Bumping version to file "${file}" with extension "${fileExtension}"`)
 
-          return handleVersioningByExtension(fileExtension, file, versionPath, recommendation.releaseType)
+          return handleVersioningByExtension(fileExtension, file, versionPath, recommendation.releaseType, skipBump)
         })
       )
-
-      newVersion = versioning[0].newVersion
       oldVersion = versioning[0].oldVersion
+      // If we are skipping the bump, we either use the fallback version or the old version as the new version.
+      if(skipBump){
+        newVersion = fallbackVersion && !oldVersion ? fallbackVersion : oldVersion
+      } else {
+        newVersion = versioning[0].newVersion
+      }
```

Comment:
Is this needed as `handleVersioningByExtension` makes sure the correct version is set in `newVersion`?

Reply:
Yeah, that's a good point, I think I should move the entire versioning logic into that function instead of putting it here